proposals/mips/mip-b53/mip-b53.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import "@forge-std/Test.sol";
+
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import {MTokenInterface} from "@protocol/MTokenInterfaces.sol";
+import {MWethOwnerWrapper} from "@protocol/MWethOwnerWrapper.sol";
+
+import {HybridProposal, ActionType} from "@proposals/proposalTypes/HybridProposal.sol";
+import {AllChainAddresses as Addresses} from "@proposals/Addresses.sol";
+import {BASE_FORK_ID} from "@utils/ChainIds.sol";
+import {ProposalActions} from "@proposals/utils/ProposalActions.sol";
+
+import {DeployMWethOwnerWrapper} from "@script/DeployMWethOwnerWrapper.s.sol";
+
+/// @title MIP-B53: WETH Market Ownership Wrapper
+/// @notice Proposal to deploy and migrate WETH market admin to a wrapper contract
+///         that can reliably receive native ETH, enabling reserve reductions.
+/// @dev This proposal:
+///      1. Deploys MWethOwnerWrapper implementation and proxy
+///      2. Transfers WETH market admin from TEMPORAL_GOVERNOR to the wrapper
+///      3. Wrapper is owned by TEMPORAL_GOVERNOR, maintaining governance control
+///      4. Enables future WETH reserve reductions via the wrapper
+contract mipb53 is HybridProposal {
+    using ProposalActions for *;
+
+    string public constant override name = "MIP-B53";
+
+    constructor() {
+        bytes memory proposalDescription = abi.encodePacked(
+            vm.readFile("./proposals/mips/mip-b53/b53.md")
+        );
+        _setProposalDescription(proposalDescription);
+    }
+
+    function primaryForkId() public pure override returns (uint256) {
+        return BASE_FORK_ID;
+    }
+
+    function deploy(Addresses addresses, address) public override {
+        vm.selectFork(BASE_FORK_ID);
```

Comment:
dumb question: we're only deploying this to Base? there's an ETH core market on OP as well